packages/terra-form-select/src/SelectField.jsx

```diff
@@ -140,6 +146,8 @@ const SelectField = ({
     isInline={isInline}
     isLabelHidden={isLabelHidden}
     htmlFor={selectId}
+    maxWidth={maxWidth}
+    {...customProps}
```

Comment:
Why was the customProps spread moved to last?

Reply:
Oh yeah, was testing order to max sure I didn't screw up priority.